specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-04-01-preview/authorization-RoleAssignmentsCalls.json

```diff
@@ -390,6 +396,12 @@
               "$ref": "#/definitions/RoleAssignment"
             }
           },
+          "500": {
```

Comment:
>"500" [](start = 10, length = 5)

Curious why you defined 500 response model. Was there a requirement?

Reply:
In some cases we return a message in a format that is not compatible with the default response schema